packages/contracts-bedrock/test/kontrol/proofs/OptimismPortal.k.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import { DeploymentSummary } from "./utils/DeploymentSummary.sol";
+import { KontrolUtils } from "./utils/KontrolUtils.sol";
+import { Types } from "src/libraries/Types.sol";
+import {
+    IOptimismPortal as OptimismPortal,
+    ISuperchainConfig as SuperchainConfig
+} from "./interfaces/KontrolInterfaces.sol";
+
+contract OptimismPortalKontrol is DeploymentSummary, KontrolUtils {
+    OptimismPortal optimismPortal;
+    SuperchainConfig superchainConfig;
+
+    function setUp() public {
+        recreateDeployment();
+        optimismPortal = OptimismPortal(payable(OptimismPortalProxyAddress));
+        superchainConfig = SuperchainConfig(SuperchainConfigProxyAddress);
+    }
+
+    /// TODO: Replace struct parameters and workarounds with the appropriate
```

Comment:
I think this PR, cc @JuanCoRo to confirm https://github.com/runtimeverification/kontrol/pull/254

Reply:
Correct! I've created this more general issue (https://github.com/runtimeverification/kontrol/issues/272) since we're pursuing other routes as well. I'll be adding it to the code